pkg/config/validate.go

```diff
@@ -109,20 +110,22 @@ func (dc DeploymentConfig) executeValidators() error {
 	}
 
 	if warned || errored {
-		log.Println("validator failures can indicate a credentials problem.")
-		log.Println("cloud credentials associated with your Google Cloud account can be generated by the following command:")
-		log.Println("  $ gcloud auth application-default login")
-		log.Println("troubleshooting info appears at:")
-		log.Println("https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/README.md#supplying-cloud-credentials-to-terraform")
-		log.Println()
-		log.Println("validation can be configured:")
-		log.Println("- treat failures as warnings by using the create command")
-		log.Println("  with the flag \"--validation-level WARNING\"")
-		log.Println("- can be disabled entirely by using the create command")
-		log.Println("  with the flag \"--validation-level IGNORE\"")
-		log.Println("- a custom set of validators can be configured following")
-		log.Println("  instructions at:")
-		log.Println("https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/docs/blueprint-validation.md")
+		log.Println("One or more blueprint validators has failed. See messages above for suggested")
+		log.Println("actions. General troubleshooting guidance and instructions for configuring")
+		log.Println("validators are shown below.")
+		log.Println("")
+		log.Println("- https://goo.gle/hpc-toolkit-troubleshooting")
```

Comment:
I will just note here that I recognize you are using the short links. On one hand this looks cleaner, on the other there is an added dependency on the maintenance of the short links. I could go either way but am fine to use them.

Reply:
I suggest trying them and reverting if it becomes a problem.